TextRecognitionDataGenerator/data_generator.py

```diff
@@ -119,15 +119,11 @@ def generate(cls, index, text, font, out_dir, size, extension, skewing_angle, ra
         #####################################
         # Generate name for resulting image #
         #####################################
-        if name_format == 0:
-            image_name = '{}_{}.{}'.format(text, str(index), extension)
-        elif name_format == 1:
-            image_name = '{}_{}.{}'.format(str(index), text, extension)
-        elif name_format == 2:
-            image_name = '{}.{}'.format(str(index),extension)
-        else:
-            print('{} is not a valid name format. Using default.'.format(name_format))
-            image_name = '{}_{}.{}'.format(text, str(index), extension)
-
+ 
+        image_name = '{}.{}'.format(str(uuid.uuid4()),extension)
```

Comment:
Was this added to the PR by mistake? I can't really change the behavior of TRDG like that.

Reply:
yes, sorry, this was added by mistake, I was working on make TRDG available for amharic language and it printed blank images, so I made changes which involved adding a new font for amharic & removing the name factor as utf-8 based file names were saved as "", which happened to clash with my own configuration. i hope it didn't cause any harm.